### PR TITLE
add publish husky hooks command

### PR DESCRIPTION
**Pull request by ArielMejiaDev**
## How it adds value

- This PR reduces developer's time as it installs the required libraries and configures **husky git hooks** automatically for **lint** or **fix and commit** strategies

- If a developer does not have experience with the last **Husky** version, it reduces time to look up to figure out how **Husky** and **staged-lint** work together and configure it

- If a project does not have a **git repository initialized husky** will not catch any commit and **duster** will not be executed anytime, this PR prevents this by initializing/reinitializing the git repository, it seems safe [git init manual page](https://mirrors.edge.kernel.org/pub/software/scm/git/docs/git-init.html)

- When [Duster was announced](https://tighten.com/insights/announcing-duster-a-laravel-code-linter-and-fixer/) **Laravel's** `package.json` file had an `npm test` script, now it does not have it when **duster** runs with **staged-lint** it throws an exception `Missing script: "test"`, this PR  handles this automatically for you adding the **stage-lint** config file without this line.

### CODE REVIEW

**Comment by ArielMejiaDev** [comment]
It would be nice to add a trait to handle `success` method in `GitHubActionsCommand` and `HuskyHookCommand`, if there is any suggestion about where to place the trait I would be glad :) ... pretty excited about this PR 🤩

File: .idea/.gitignore
Comment (driftingly) [inline .idea/.gitignore:1]: Can you remove all these `.idea` files?

**Review by driftingly — CHANGES_REQUESTED** [review]
This looks really good, thanks 🙌 
Just a bit of files to clean up.

**Comment by mattstauffer** [comment]
@ArielMejiaDev You asked for me to review this; I don't FULLY understand without digging in further what implications this has on our relationship with Dusky, _but_ the code looks great and i fully trust @driftingly's review, so great work! Please remove these `.idea` files and then Anthony will merge when ready--thanks for the PR!

**Review by driftingly — APPROVED** [review]

**Pull request merged**